package.json

```diff
@@ -11,7 +11,7 @@
     "docs:build": "storybook build -o public/storybook",
     "lint:eslint": "eslint src integrationTesting",
     "lint:prettier": "prettier --check src integrationTesting",
-    "lint:translations": "ts-node src/tools/lint-translations",
+    "lint:translations": "ts-node src/tools/lint-translations.ts",
```

Comment:
Is including the suffix necessary for some environments, or why this change?

Reply:
Yes, from c78cb68ec93247b9d34e406f6d780739d9ae0a21:

> Without this change, running the script in Node.js >=23 fails with "Error: Cannot find module `./lint-translations`".